contracts/atm/ATMGovernance.sol

```diff
@@ -9,17 +9,18 @@ import "../util/AddressLib.sol";
 import "../base/TInitializable.sol";
 
 // Contracts
-import "@openzeppelin/contracts-ethereum-package/contracts/access/roles/SignerRole.sol";
+import "./BaseATM.sol";
 
 // Interfaces
-import "./IATMGovernance.sol";
+import "./ATMGovernanceInterface.sol";
+import "../settings/IATMSettings.sol";
 
 
 /**
     @notice This contract is used to modify Risk Settings, CRA or DataProviders for a specific ATM.
     @author develop@teller.finance
  */
-contract ATMGovernance is SignerRole, IATMGovernance, TInitializable {
+contract ATMGovernance is BaseATM, ATMGovernanceInterface, TInitializable {
```

Comment:
Why did we remove SignerRole? weren't we using it?

Why did we change the interface name? We are trying to use the new naming convention.

Reply:
`SignerRole` was removed because we are using the modifier `onlyPauser` that checks if the sender has the pauser role from the `Settings` (`atmSettings.settings`) contract